DataFormats/PatCandidates/interface/libminifloat.h

```diff
@@ -76,6 +78,44 @@ class MiniFloatConverter {
             return conv.flt;
         }
 
+        class ReduceMantissaToNbitsRounding {
+            public:
+                ReduceMantissaToNbitsRounding(int bits) : 
+                    shift(23-bits), mask((0xFFFFFFFF >> (shift)) << (shift)), 
```

Comment:
it looks like this duplicates the functionality of ```inline static float reduceMantissaToNbitsRounding(const float &f)```
please remove the other implementation or see how the code duplication can be reduced

Reply:
is the solution in https://github.com/cms-sw/cmssw/pull/20367/commits/9f31d22ef866657ccd2f9701a6fe3061915edb48 ok for you?

Reply:
yes, it looks OK.
Thank you.